src/wasm.rs

```diff
@@ -66,7 +66,7 @@ fn format_kv_pairs(record: &Record) -> Option<Object> {
             if self.hashmap.is_none() {
                 self.hashmap = Some(HashMap::new())
             }
-            let hm = self.hashmap.as_mut()?;
+            let hm = self.hashmap.as_mut().ok_or(kv::Error::msg("No hashmap found"))?;
```

Comment:
hmmm something else is going on here, as lines [66-68](https://github.com/lrlna/femme/pull/39/files#diff-6d7bc9e99526e7f06b818a6103bd30aaa067263cc8e34a14197e1a3bcf2dd4dbR66-R68) create a hashmap if there are none. Would you be able to paste the error you get when you compile with wasm?

Reply:
Here you go:

```
# wasm-pack test --chrome --headless

[...]

error[E0277]: the `?` operator can only be used on `Result`s, not `Option`s, in a method that returns `Result`
  --> /Users/1234/.cargo/registry/src/github.com-1ecc6299db9ec823/femme-2.2.0/src/wasm.rs:69:43
   |
61 | /         fn visit_pair(
62 | |             &mut self,
63 | |             key: kv::Key<'kvs>,
64 | |             val: kv::Value<'kvs>,
...  |
69 | |             let hm = self.hashmap.as_mut()?;
   | |                                           ^ use `.ok_or(...)?` to provide an error compatible with `Result<(), log::kv::Error>`
70 | |             hm.insert(key.to_string(), val.to_string());
71 | |             Ok(())
72 | |         }
   | |_________- this function returns a `Result`
   |
   = help: the trait `FromResidual<Option<Infallible>>` is not implemented for `Result<(), log::kv::Error>`
```